### PR TITLE
Support angular 18

### PR DESCRIPTION
**Pull request by MaximBelov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,47 +38,47 @@
   "peerDependencies": {
     "@angular/core": ">=13.0.0",
     "@messageformat/core": "^3.2.0",
-    "@ngx-translate/core": "^14.0.0 || ^15.0.0"
+    "@ngx-translate/core": "^14.0.0 || ^15.0.0 || ^18.0.0"
```
Comment (lephyrus) [inline package.json:41]: v18 is Angular, this should be v16. :slightly_smiling_face:

**Review by lephyrus — CHANGES_REQUESTED** [review]
Thanks for the PR! :+1:

File: package.json (hunk 1 above)
Comment (MaximBelov) [reply package.json]: yep
fixed

**Comment by lephyrus** [comment]
I've integrated the changes here in #119.

**Pull request closed without merging**